src/Panel.tsx

```diff
@@ -60,15 +61,30 @@ export const Panel = ({ active, api }: PanelProps) => {
 
   // If the user creates a project in a dialog (either during login or later, it get set here)
   const [createdProjectId, setCreatedProjectId] = useState<Project["id"]>();
+  const [addonUninstalled, setAddonUninstalled] = useState<boolean>(false);
+
+  if (addonUninstalled) {
+    return (
+      <Provider key={PANEL_ID} value={client}>
+        <UninstallProvider>
```

Comment:
Should we fold all these providers up into a single provider?

Reply:
Yeah, I'll follow that up in subsequent PR.